### DISCUSSION TITLE
tutorials for medical imaging reconstruction and synthesis

### DISCUSSION THREAD

**Discussion by hw-ju** [Project-MONAI/tutorials#538] [Ideas]
Will MONAI Core provide tutorials for medical imaging reconstruction (i.e., using DL model to map sensor space to image space) and synthesis (e.g., map MRI to CT, there was a request last year https://github.com/Project-MONAI/tutorials/issues/100)?

**Comment by wyli** [comment]
Thanks for your interest, the plan is to include a reconstruction workflow in summer 2022.
